### ISSUE TITLE
Skipping mechanism anomaly

### ISSUE THREAD

**Issue by mateatslc** [elwin013/parcel-plugin-static-files-copy#71]
With the current `v2.5.0` release, given the following config arrangement...
```
"staticFiles": {
	"staticPath": [
		{
			"staticPath": "foo"
		},
		{
			"staticPath": "bar"
		}
	]
}
```
...where `foo` is a non-existent folder, plugin wont copy stuff from `bar`.

Same works fine in `v2.4.4`: `foo` gets skipped, content of `bar` gets copied as expected.

**Comment by mateatslc** [comment]
Same with this simpler format...
```
"staticFiles": {
	"staticPath": [
		"foo",
		"bar"
	]
}
```

**Comment by elwin013** [comment]
Hey @mateatslc! Thank you for spotting this. Fixed in 2.5.1 - just published to npmjs.
